nexus/reconfigurator/planning/src/blueprint_editor/sled_editor.rs

```diff
@@ -587,6 +588,9 @@ impl ActiveSledEditor {
                     .remove_mupdate_override
                     .finalize(),
                 host_phase_2: self.host_phase_2.finalize(),
+                // TODO this will come in a subsequent PR
+                measurements:
+                    BlueprintMeasurementsDesiredContents::default_contents(),
```

Comment:
I wonder if we could better track these call sites with a separate constructor, like `placeholder()`?  Instead of relying on these TODOs?

(I just suggested something similar to Karen on another PR)